src/cfml/commands/server/toggle.cfc

```diff
@@ -0,0 +1,65 @@
+/**
+ * Toggle an embedded CFML server.  Run command from the web root of the server, or use the short name.
+ * .
+ * {code:bash}
+ * server toggle
+ * server toggle --!open
+ * server toggle serverName
+ * {code}
+ **/
+component extends="commandbox.system.BaseCommand" aliases="toggle" excludeFromHelp=false {
+
+	// DI
+	property name="serverService" inject="ServerService";
+
+	/**
+	 * @name.hint the short name of the server to stop
+	 * @name.optionsUDF serverNameComplete
+	 * @directory.hint web root for the server
+     * @open.hint    	open a browser after starting
+	 **/
+	function run(string name="", string directory="", boolean open=true) {
+
+		// Discover by shortname or webroot and get server info
+		var servers = { id: serverService.getServerInfoByDiscovery(
+			directory 	= arguments.directory,
+			name		= arguments.name
+		) };
+
+		// Verify server info
+		if( structIsEmpty( servers.id ) ){
+			error( "The server you requested to toggle was not found (webroot=#arguments.directory#, name=#arguments.name#)." );
+			print.line( "You can use the 'server list' command to get all the available servers." );
+			return;
+		}
+
+		// Toggle the server(s)
+		for( var id in servers ) {
+			var serverInfo = servers[ id ];
+
+			if( serverInfo.status == 'stopped' ) {
+                print.yellowLine( 'Starting ' & serverInfo.name & '...' ).toConsole();
+                return serverService.start(
```

Comment:
We might want to consider running the start and stop commands directly here via `runCommand()`.  There might be some logic related to grabbing `box.json` defaults in the commands.  That probably means we should move that logic into the server, but it's probably worth checking.  A simple test would be to toggle a stopped server and see if it starts using the defaultPort in your box.json.